### PR TITLE
Aggregation Command - Get User Data - V2

### PR DESCRIPTION
**Pull request by mmhw**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Related: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-11252)

## Description
Aggregation Command - Get User Data

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonScripts/ReleaseNotes/1_15_67.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### New: get-user-data
+
+- New: This script gathers user data from multiple integrations and returns an Account entity with consolidated information to the context.<~XSOAR> (Available from Cortex XSOAR 6.10.0).</~XSOAR>
```
Comment (ShirleyDenkberg) [inline Packs/CommonScripts/ReleaseNotes/1_15_67.md:6]:
```suggestion
New: This script gathers user data from multiple integrations and returns an Account entity with consolidated information to the context.<~XSOAR> (Available from Cortex XSOAR 6.10.0).</~XSOAR>
```

File: Packs/CommonScripts/Scripts/GetUserData/GetUserData.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
+args:
+- description: List of users IDs of the user to retrieve.
+  name: user_id
+  isArray: true
+- description: List of users names of the user to retrieve.
```
Comment (ShirleyDenkberg) [inline Packs/CommonScripts/Scripts/GetUserData/GetUserData.yml:5]:
```suggestion
- description: List of names of the users to retrieve.
```
Should this be:  A comma-separated list of names of the users to retrieve?

File: Packs/CommonScripts/Scripts/GetUserData/GetUserData.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  name: user_name
+  isArray: true
+- description: List of users emails of the user to retrieve.
```
Comment (ShirleyDenkberg) [inline Packs/CommonScripts/Scripts/GetUserData/GetUserData.yml:8]:
```suggestion
- description: List of emails of the users to retrieve.
```
Should this be: A comma-separated list of emails of the users to retrieve?

File: Packs/CommonScripts/Scripts/GetUserData/GetUserData.yml [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  name: user_email
+  isArray: true
+- description: The domain to retrieve users from, available only for the iam-get-user command.
```
Comment (ShirleyDenkberg) [inline Packs/CommonScripts/Scripts/GetUserData/GetUserData.yml:11]:
```suggestion
- description: The domain to retrieve users from. Available only for the iam-get-user command.
```

File: Packs/CommonScripts/Scripts/GetUserData/GetUserData.yml [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  - 'SailPointIdentityNow'
+  - 'Active Directory Query v2'
+  - 'PingOne'
+  - 'Okta v2'
+  - 'AWS - IAM'
+  - 'Microsoft Graph User'
+  - 'SailPointIdentityIQ'
+  - 'Cortex XDR - IR'
+  - 'iam-get-user'
+- description: Whether to retrieve human readable entry for every command or only the final result.
```
Comment (ShirleyDenkberg) [inline Packs/CommonScripts/Scripts/GetUserData/GetUserData.yml:28]:
```suggestion
- description: Whether to retrieve human readable entry for every command or only the final result. True means to retrieve human readable entry for every command. False means to human readable only for the final result.
```

File: Packs/CommonScripts/Scripts/GetUserData/README.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+---
+
+| **Argument Name** | **Description** |
+| --- | --- |
+| user_id | List of users IDs of the user to retrieve. |
+| user_name | List of users names of the user to retrieve. |
+| user_email | List of users emails of the user to retrieve. |
+| domain | The domain to retrieve users from, available only for the iam-get-user command. |
+| brands | Which integrations brands to run the command for. If not provided, the command will run for all available integrations.<br/>For multi-select provide a comma-separated list. For example: "SailPointIdentityNow,Active Directory Query v2,PingOne". |
+| verbose | Whether to retrieve human readable entry for every command or only the final result. |
```
Comment (ShirleyDenkberg) [inline Packs/CommonScripts/Scripts/GetUserData/README.md:23]:
```suggestion
| user_id | List of IDs of the user to retrieve. |
| user_name | List of names of the users to retrieve. |
| user_email | List of emails of the users to retrieve. |
| domain | The domain to retrieve users from. Available only for the iam-get-user command. |
| brands | Which integrations brands to run the command for. If not provided, the command will run for all available integrations.<br/>For multi-select provide a comma-separated list. For example: "SailPointIdentityNow,Active Directory Query v2,PingOne". |
| verbose | Whether to retrieve human readable entry for every command or only the final result.  True means to retrieve human readable entry for every command. False means to human readable only for the final result. |
```

**Comment by ShirleyDenkberg** [comment]
@anara123 @bziser Doc review completed.

**Review by DeanArbel — APPROVED** [review]
Looks good, amazing job on this

**Pull request merged**